use-cases/using-SRIOV-and-kata.md

```diff
@@ -316,3 +317,33 @@ The following example launches a Kata Containers container using SR-IOV:
    ```
    sriov-2:~$ sudo docker run --runtime=kata-runtime --net=vfnet --cap-add SYS_ADMIN --ip=192.168.0.11 -it mcastelino/iperf iperf3 -c 192.168.0.10 bash -c "mount -t ramfs -o size=20M ramfs /tmp; iperf3 -c 192.168.0.10"
    ```
+## Example 2: Launch a Kata container using SR-IOV without SR-IOV Plugin
+
+The second example launches a Kata container using SR-IOV, however doesn't needs an additional plugin.
```

Comment:
Suggest changing "doesn't" to "does not"

Reply:
@HolyShitMan - it looks like this one tiny change is maybe the only thing stopping this getting merged - any chance you can update for us? thx!

Reply:
@klynnrif The change you requested has been made, I believe, and it presently blocks the merge. Could you please check? Thanks.